src/frontend-samples/explode-sample/ExplodeTile.ts

```diff
@@ -586,19 +586,12 @@ export class ExplodedGraphicsTile extends Tile {
     assert(TileFormat.IModel === format);
 
     const tree = this.tree;
-    const reader = ImdlReader.create(stream, tree.iModel, tree.modelId, tree.is3d, system, undefined, undefined, isCanceled, undefined, this.contentId);
-
-    let content: TileContent = { isLeaf: true };
-    if (reader) {
-      try {
-        content = await reader.read();
-      } catch (_) {
-        //
-      }
-    }
+
+    // When for next iModel.js update, the API will include an argument for batching options.  This should be set to the contentId.
+    const graphic = await readElementGraphics(data, tree.iModel, tree.modelId, tree.is3d);
 
     (this.tree as ExplodeTileTree).elementContentLoaded(this.data.elementId);
-    return content;
+    return { isLeaf: true, graphic } as TileContent;
```

Comment:
Remove `as TileContent`; it bypasses the type-checker.

Reply:
TileContent cast removed.